daisy/common.go

```diff
@@ -39,6 +39,11 @@ func getUser() string {
 	return "unknown"
 }
 
+// NamedSubexp extracts sub matches in the exp
+func NamedSubexp(re *regexp.Regexp, s string) map[string]string {
```

Comment:
Just make the other function public (youll have more code to update but it beats having two functions)